Internal patch to disable the image_source (Create Volume -> Volume
Source -> image) because copy_volume_from_image is not supported yet.
This patch will not be committed upstream.

--- horizon-9.0.1/openstack_dashboard/dashboards/project/volumes/volumes/forms.py.~1~	2016-06-02 13:05:51.000000000 -0700
+++ horizon-9.0.1/openstack_dashboard/dashboards/project/volumes/volumes/forms.py	2016-06-29 23:54:56.869785260 -0700
@@ -230,6 +230,9 @@ class CreateForm(forms.SelfHandlingForm)
 
         images = utils.get_available_images(request,
                                             request.user.tenant_id)
+        # Disable the image source on Solaris
+        images = False
+        
         if images:
             source_type_choices.append(("image_source", _("Image")))
             choices = [('', _("Choose an image"))]
